authors/settings.py

```diff
@@ -45,7 +45,10 @@
     'authors.apps.authentication',
     'authors.apps.core',
     'authors.apps.profiles',
+
+
```

Comment:
these spaces seem unnecessary. I would get rid of them

Reply:
resolved